js/main.js

```diff
@@ -0,0 +1,177 @@
+/* global google */
+
+/**
+ * Tag constructor
+ *
+ * @constructor
+ * @param {string} searchTerm The tag to search for
+ */
+var Tag = function(searchTerm) {
+  this.searchTerm = searchTerm;
+};
+
+/**
+ * Get the top places where this tag was used
+ *
+ * @param {function} callback An optional method to call upon completion
+ */
+Tag.prototype.getPlaces = function(callback) {
+  $.ajax({
+    url: "http://api.flickr.com/services/rest/?jsoncallback=?",
+    dataType: 'jsonp',
+    data: {
+      method: 'flickr.places.placesForTags',
+      api_key: 'b3a7fa326b8255e6af7900b7a58841e0',
```

Comment:
When you have shared configuration like this, you might want to move it to a single property/constant so that it can be updated in one place.